mapID_to_collection_DB/osu_collector_dumper.py

```diff
@@ -1,107 +1,66 @@
 import requests
-import json
 import re
 import time
 import subprocess
 
-def osu_collector_dump(collection_id, collection_path):
-    
-    collection_path_no_extension_1 = re.sub(".*(\\\|\\\\)", "", collection_path)
-    collection_path_no_extension_2 = re.sub("\..*", "", collection_path_no_extension_1)
-    
-    collection_id_regex = re.search("\d*$", collection_id)
-    
-    hasMore = True
-    filepath = "list.txt"
-    csv_filepath = 'CollectionCSVtoDB\\' + collection_path_no_extension_2 + '.csv'
-
-    with open(filepath, 'w') as id_dump:
-        id_dump.close()
-
-    with open(csv_filepath, 'w') as id_dump:
-        id_dump.close()
-    
-    if hasMore == True:
-
-        url = "https://osucollector.com/api/collections/" + collection_id_regex.group(0)
-
-        r = requests.get(url)
-        collection = json.loads(r.text)
-
-        print ("osu!Collectior: waiting.")
-
-        regex_filtered = re.findall('(?<="id": )\d{1,8}', json.dumps(collection))
-
-        hashes_regex_filtered = re.findall('(?<="checksum": ").{32}', json.dumps(collection))
-        
-        for item in regex_filtered:
-            with open (filepath, "a") as id_dump:
-                id_dump.writelines([item])
-                id_dump.writelines(["\n"])
 
-        for item in hashes_regex_filtered:
-            with open (csv_filepath, "a") as hash_dump:
-                    hash_dump.writelines([",,"])
-                    hash_dump.writelines([item])
-                    hash_dump.writelines(["\n"])
```

Comment:
This method of dumping should be kept in some way. The api call used to grab a collection with no sorting is much faster than grabbing one with sorting when collections have more than 100 maps. 
The following api call grabs all maps in the collection at once regardless of how many maps are in the collection instead of grabbing 100 maps per api call:
```
https://osucollector.com/api/collections/{collection_id}
```

This method is solely for when there is no sorting and when sorting is required it cannot be used.

Reply:
This should be resolved by 9c21d41